docs/library/sys.rst

```diff
@@ -15,6 +15,21 @@ Functions
    function raise as `SystemExit` exception. If an argument is given, its
    value given as an argument to `SystemExit`.
 
+.. function:: print_exception(exc, file=sys.stdout, /)
+
+   Print exception with a traceback to a file-like object *file* (or
+   `sys.stdout` by default).
+
+   .. admonition:: Difference to CPython
+      :class: attention
+
+      This is simplified version of a function which appears in the
+      ``traceback`` module in CPython. Unlike ``traceback.print_exception()``,
+      this function takes just exception value instead of exception type,
+      exception value, and traceback object; *file* argument should be
+      positional; further arguments are not supported. CPython-compatible
+      ``traceback`` module can be found in `micropython-lib`.
```

Comment:
https://docs.circuitpython.org/en/latest/shared-bindings/traceback/index.html#module-traceback circuitpython has a built in traceback module

Reply:
Circuitpython seems to have a sys.print_exception(), traceback.print_exception() and traceback.format_exception() functions. I suspect inherited from micropython. My concern was that the sys.print_exception() was included in the circuitpython documentation.

Reply:
oops, make that "was not included" :grin: